### DISCUSSION TITLE
Unable to add custom integration

### DISCUSSION THREAD

**Discussion by brixham99** [alandtse/tesla#28] [Q&A] [answered]
I had been using the Tesla Core integration, which has now stopped working for me.  Having removed it, I'm now struggling to get this custom integration working.  Not sure if I'm doing something wrong here, but after following the installation steps, adding this Repo into HACS, when I try to add the integration, searching for Tesla Custom doesn't get any results.  I went to try the method of manually copying the files under custom_components, but the tesla_custom folder is already there.  Am I missing something?

**Comment by brixham99** [comment, marked as answer]
Eventually it popped up in discovered integrations, however the captcha doesn't seem to work, it shows:

ERROR for site owner:
Invalid domain for site key

Any ideas?

**Reply by alandtse** [reply]
https://github.com/alandtse/tesla/issues/36
